internal/backends/gl/Cargo.toml

```diff
@@ -21,10 +21,13 @@ svg = ["resvg", "usvg", "tiny-skia"]
 wayland = ["winit/wayland", "glutin/wayland", "copypasta/wayland"]
 x11 = ["winit/x11", "glutin/x11", "copypasta/x11"]
 
-default = ["svg"]
+rtti = ["i-slint-core/rtti"]
+
+default = ["svg", "rtti"]
```

Comment:
(unrelated to this patch: I thin rtti shouldn't be enabled by default, also in the Qt backend)